cmd/xl-v1-object.go

```diff
@@ -249,7 +249,11 @@ func (xl xlObjects) GetObject(bucket, object string, startOffset int64, length i
 	chunkSize := getChunkSize(xlMeta.Erasure.BlockSize, xlMeta.Erasure.DataBlocks)
 	pool := bpool.NewBytePool(chunkSize, len(onlineDisks))
 
-	// Read from all parts.
+	storage, err := NewErasureStorage(onlineDisks, xlMeta.Erasure.DataBlocks, xlMeta.Erasure.ParityBlocks, xl.readQuorum, xl.writeQuorum)
```

Comment:
We are actually remove xl.readQuorum from the xlObject struct altogether -- and instead always use xlMeta.Erasure.DataBlocks as read quorum for GetObject operations.